lib/ontologies_linked_data/models/ontology_submission.rb

```diff
@@ -471,11 +473,11 @@ def generate_rdf(logger, reasoning: true)
               logger.info("error deleting owlapi.rdf")
             end
           end
-          owlapi = owlapi_parser(logger: nil)
-
-          if !reasoning
-            owlapi.disable_reasoner
-          end
+          owlapi = LinkedData::Parser::OWLAPICommand.new(
```

Comment:
maybe there is a downgrade in the code here, why remove the usage of the helper `owlapi_parser(logger: nil)` and replaced with  `LinkedData::Parser::OWLAPICommand.new` ?

Reply:
@mdorf and @alexskr - I have the same question as Syphax here. Was this change intended?

Reply:
@syphax-bouazzouni, I just merged the code from your earlier pull request.  From my understanding, the `owlapi_parser(logger: nil)` method includes a call to `unzip_submission(logger)`, which is already being run inside the `process_submission` method on line 1005. That call had to be done earlier because your updated `generate_rdf` method now requires the `file_path` to be passed as an argument. So calling `owlapi_parser(logger: nil)` inside `generate_rdf` would duplicate the unzipping process. Again, I simply merged your earlier pull request and fixed any code that it had missed.

Reply:
@syphax-bouazzouni, I think we can simply remove the `file_path` from the list of required arguments for `generate_rdf` and then make a call to the `owlapi_parser(logger: nil)` method inside `generate_rdf` instead of the direct call to `LinkedData::Parser::OWLAPICommand.new`. From what I can see this is the only place in the `generate_rdf` that needs the `file_path`.

Reply:
Yes removing that argument, would be the right solution here, even in our local branch `generate_rdf` does not have the `file_path ` argument, I don't know why it ended up in my PR.

Reply:
I'll make that change on our end and re-test

Reply:
Great, thanks, and sorry for the bother.

Reply:
the refactor that @mdorf suggested will be implemented after this release is complete